packages/url-loader/src/plugins/cropping.ts

```diff
@@ -162,8 +172,8 @@ function collectTransformations(collectOptions: CropOptions) {
   // Normalize sizing parameters
 
   let gravity = collectOptions.gravity;
-  let height = normalizeNumberParameter(collectOptions.height);
-  let width = normalizeNumberParameter(collectOptions.width);
+  const height = normalizeNumberParameter(collectOptions.height);
```

Comment:
question: does this work? I mean, are you able to modify the height and gravity variables if they are changed to consts?

Reply:
This was actually based on an eslint rule I enabled to catch cases where `let` is not needed because the variable is never reassigned:

https://eslint.org/docs/latest/rules/prefer-const

Reply:
Ahh sorry, I just see now that gravity is still a let. Closing the comment!